sql/wsrep_sst.cc

```diff
@@ -1132,13 +1133,15 @@ int wsrep_remove_sst_user(bool initialize_thread) {
   // --skip-grant-tables option. It would fail enyway with error.
   // This will prevent writing out error to error log.
   if (skip_grant_tables()) {
-      return ECANCELED;
+    return ECANCELED;
```

Comment:
do we still need wsrep_remove_sst_user() function? It has been removed from sql_parse.cc shutdown() and from sst_donor_thread(). Any other callers?

Reply:
I readded the sst user removal to the end of the donor thread, as it's safe there. This means that with normal code flow, the sst user will always be removed once sst is done, only the extra removal from server shutdown is gone.